src/intro/hello-world.md

```diff
@@ -279,6 +282,7 @@ Let's break this down.
 ```admonish warning title="Correct node type"
 When adding an instance of your `Player` class to the scene, make sure to select node type `Player` **and not its base `Sprite2D`**.
 Otherwise, your Rust logic will not run.
+We will guide you to make that change to your scene later, when you're ready to test it.
```

Comment:
This doesn't add much, I'd remove it.

We should keep warning boxes as short and to-the-point as possible, otherwise they become inflationary. Warning boxes explicitly don't follow the chronological flow of the tutorial.